src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstBase;
+    private int secondBase;
+    private int height;
+
+    public IsoscelesTrapezoid(Color color, int firstBase, int secondBase, int height) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
+    }
+
+    public double getArea() {
+        return (firstBase + secondBase) / 2.0 * height;
```

Comment:
The `getArea` method should have a single purpose. Since the class `IsoscelesTrapezoid` also has a `draw` method, consider separating the drawing behavior from the area calculation by creating different interfaces or abstract classes if needed.